IFC4x3/Properties/o/OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OverallDiameter_3583UJ6cD6cBUOPJ6Z7hRk" Name="OverallDiameter" UniqueId="c5203793-1a63-4698-b798-6531a31eb6ee" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Should this use common Nominal Diameter Property instead?

Reply:
If possible, we should harmonize the other relevant properties with a generic description.  Typically we picked the first one and changed the description.